libs/ui-toolkit/src/components/icon/vega-icons/svg-icons/icon-vote.tsx

```diff
@@ -0,0 +1,12 @@
+export const IconVote = ({ size = 16 }: { size: number }) => {
+  return (
+    <svg width={size} height={size} viewBox="0 0 24 24">
```

Comment:
Can you clean this icon up? I doubt the fillRule and clipRule are needed. Also all viewboxes for these icons should be 0 0 16 16 so we can get consistent behavior when sizing them.

Reply:
The fillrule and cliprule are needed for the internal 'tick' inside the icon. I'll resize it if it bothers you, but it'll make no different - the viewbox only determines the relative size for the coordinates. The 'width' and 'height' properties are what will determine the rendered size. Also note there are other icons in the set that aren't set to a viewbox of the same size.